otherlibs/unix/access.c

```diff
@@ -49,14 +51,15 @@ static int access_permission_table[] = {
 CAMLprim value unix_access(value path, value perms)
 {
   CAMLparam2(path, perms);
-  char * p;
+  char_os * p;
   int ret, cv_flags;
 
   caml_unix_check_path(path, "access");
   cv_flags = caml_convert_flag_list(perms, access_permission_table);
-  p = caml_stat_strdup(String_val(path));
+  p = caml_stat_strdup_to_os(String_val(path));
   caml_enter_blocking_section();
-  ret = access(p, cv_flags);
+  fflush(stdout);
```

Comment:
Was this call to `fflush` left over from some debugging code?

Reply:
Man, that's embarrassing, thanks for spotting that. If you can believe it, I do review my diffs...